modules/core/05-port/types/module.go

```diff
@@ -119,15 +119,13 @@ type UpgradableModule interface {
 		version, previousVersion string,
 	) (string, error)
 
-	// OnChanUpgradeTry TODO
-	OnChanUpgradeTry(
+	// OnUpgradeBlock
```

Comment:
update godoc to be descriptive. This is the try step, so we can indicate that the counterparty upgrade has been verified and we have set our own upgrade